hal/src/boron/cellular_hal.cpp

```diff
@@ -221,6 +223,41 @@ int cellular_credentials_clear(void* reserved) {
     return 0;
 }
 
+cellular_result_t cellular_global_identity(CellularGlobalIdentity* cgi_, void* reserved_)
+{
+    cellular_result_t result;
+    CellularGlobalIdentity cgi;
+
+    // Acquire Cellular NCP Client
+    const auto mgr = cellularNetworkManager();
+    CHECK_TRUE(mgr, SYSTEM_ERROR_UNKNOWN);
+    const auto client = mgr->ncpClient();
+    CHECK_TRUE(client, SYSTEM_ERROR_UNKNOWN);
+
+    // Validate Argument(s)
+    if (nullptr == cgi_) {
+        (void)reserved_;
+        result = SYSTEM_ERROR_INVALID_ARGUMENT;
+    }
+    // Load cached data into result struct
```

Comment:
Nitpicking: I find comments between code blocks confusing, but let's at least settle on one way or the other as https://github.com/particle-iot/device-os/pull/1759/files#diff-0db0ed911a9860ed2a4f0420a4dd3eefR244 does the opposite.

Reply:
There is a difference between the two. The comment above the block describes the attempted action, while the comment inside the block describes the error and its handling.